Tests/TestUtilities/ParseException.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * SonarQube Scanner for MSBuild
+ * Copyright (C) 2015-2017 SonarSource SA and Microsoft Corporation
+ * mailto: contact AT sonarsource DOT com
+ *
+ * Licensed under the MIT License.
+ * See LICENSE file in the project root for full license information.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+using System;
+using System.Runtime.Serialization;
+
+namespace TestUtilities
+{
+    /// <summary>
+    /// An exception thrown by <see cref="JavaPropertyReader"/> when parsing
+    /// a properties stream.
+    /// </summary>
+    /// Copied from https://github.com/Kajabity/Kajabity-Tools
+    public class ParseException : System.Exception
```

Comment:
Incomplete implementation of an Exception in C#. Please refer to this [guide](https://msdn.microsoft.com/en-us/library/ms173163.aspx#Anchor_1)